### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by raihan2bd**
# Vet Clinic Database 🐻 
Hello
### Hi there 👋, In this PR:
- [ ] I Created a database called vet_clinic ✅ 
- [ ]  Then created a table called animals using the project-given SQL Language ✅ 

- [ ] Then populate the given data inside the animal's table ✅ 
insert_animal_data

- [ ] Then added the necessary query to select the required data from the database ✅ 
screenshot1
screenshot2
screenshot3
screenshot4
screenshot5
screenshot6
screenshot7
screenshot8

- [ ] Last but not least I added the valid MIT License file and update the Readme file to make it professional ✅ 

***If you have any questions feel free to ask 🙂 Thank for visiting this pull request 👍***

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
```
Comment (gemmen29) [inline queries.sql:1]: - [x] Kindly take a screenshot of the result of each query as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Take%20a%20screenshot%20of%20the%20results%20of%20your%20queries.). There are some queries I wasn't able to view its result in the screenshots.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @raihan2bd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (raihan2bd) [reply queries.sql:1]: @gemmen29 Thanks a lot for your valuable suggestions. 👍  I will update this project before the next review 🙂

**Review by gemmen29 — APPROVED** [review]
Hi @raihan2bd, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**